### ISSUE TITLE
[Documentation] Add links to ember-cli-output and ember-addon-output

### ISSUE THREAD

**Issue by kellyselden** [ember-cli/ember-cli#3168]
I have been maintaining [ember-cli-output](https://github.com/kellyselden/ember-cli-output) and [ember-addon-output](https://github.com/kellyselden/ember-addon-output) since 0.0.41. They are change history repos for the output of `ember new` and `ember addon`. I use them for checking my work when I am `ember init`ing my projects after a new version, and also for quickly checking when something changed in the past.

They have been pretty helpful for me, and I would just like to offer them to anyone else that might get use out of them. I have noticed the new "changes required for updating" section of the changelog. Perhaps links to these repos could supplement that section, or maybe a reference on the readme. I can do a PR if there is interest.

**Comment by stefanpenner** [comment]
It sounds like you have found something useful, (maybe im just tired) but i am unsure what it is :P Can ELI5 it?

**Comment by kellyselden** [comment]
Those repos have one commit per ember-cli version update. They keep track of the changes to fresh projects for each new version.

**Comment by stefanpenner** [comment]
@kellyselden sure, sounds like a handy addition.
